adminlistener.go

```diff
@@ -9,8 +9,8 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/lucas-clemente/quic-go"
-	"github.com/lucas-clemente/quic-go/http3"
+	"github.com/quic-go/quic-go"
```

Comment:
Why was this library chosen over the previous one

Reply:
It's the same library, they just changed its package name.

Reply:
See https://github.com/quic-go/quic-go/issues/3611 for more information